docs/en/events.rst

```diff
@@ -79,3 +79,11 @@ The events you can listen for are as follows:
   in implementing projects or extending packages. See
   `Package Intersphinx, provided by the TYPO3 Documentation Team <https://github.com/TYPO3-Documentation/intersphinx>`__
   for an example implementation.
+- ``PreDocumentRenderEvent``: This event is called in the DocumentNodeRenderer
```

Comment:
I'd either use "in `DocumentNodeRendererer`" (with code markup), or "in the document node renderer" here and in similar cases below.